hicexplorer/hicAggregateContacts.py

```diff
@@ -71,11 +88,30 @@ def parse_arguments(args=None):
                            choices=['total-counts', 'z-score', 'obs/exp', 'none'],
                            default='none')
 
-    parserOpt.add_argument('--avgType',
-                           help='Type of average used in the output matrix. Options are mean and median. Default is median.',
-                           choices=['mean', 'median'],
+    parserOpt.add_argument('--operationType',
+                           help='Type of the operation to be applied to summerize '
+                           'the submatrices into a single matrix. Options are sum, '
+                           'mean and median. Default is median.',
+                           choices=['sum', 'mean', 'median'],
                            default='median')
 
+    parserOpt.add_argument('--perChr',
+                           help='if set, it generates a plot per chromosome. It is only affected if '
+                           'intera-chromosomal contacts are of interest.',
+                           action='store_true',
+                           required=False)
+
+    parserOpt.add_argument('--largeRegionsOperation',
+                           help='If a given coordinate in the bed file is larger than '
+                           'a bin of the input matrix, by default only the first bin '
+                           'is taken into account. However there are more posibilities '
+                           'to handel such a case. Users can ask for the last bin or '
+                           'for center of the region. As an example if a region falls into bins [4,5,6] '
+                           'and `--numberOfBins = 2` then if first, bins [3,4,7] are kept. '
+                           'If last: [3,6,7] and if center: [3,5,7].',
```

Comment:
If I read the code, it seems to me that it would be first: [3,4,5], last: [5, 6, 7] and center [4,5,6]
"mat_to_append = ma.matrix[bin_id1 - M_half:bin_id1 + M_half + 1, :][:, bin_id2 - M_half:bin_id2 + M_half + 1]"